src/mainboard/clevo/tgl-u/variants/ns50mu/devicetree.cb

```diff
@@ -0,0 +1,365 @@
+chip soc/intel/tigerlake
```

Comment:
Can we make a basebaord devicetree and add overrides here since we have more than one variant? overrides should be significantly smaller and avoid duplication of the code

Reply:
Yes, we can do this. I was planning to first develop a devicetree for NS50MU based on the NV41MZ devicetree and then find the common parts and move them to a baseboard devicetree.

I wanted to create a basic structure in this PR and then do the actual port in https://github.com/Dasharo/coreboot/pull/84